module/canvas/token.mjs

```diff
@@ -127,7 +112,7 @@ export default class Token5e extends Token {
   /** @inheritDoc */
   _onUpdate(data, options, userId) {
     super._onUpdate(data, options, userId);
-    if ( !CONFIG.Token.ringClass.enabled || (game.release.generation > 11) ) return;
+    if ( (game.release.generation >= 12) || !CONFIG.Token.ringClass.enabled ) return;
```

Comment:
Are these not equivalent?

Reply:
I just reversed it because `CONFIG.Token.ringClass.enabled` doesn't exist in V12. But it doesn't make a difference here.